crates/dkg-cli/src/main.rs

```diff
@@ -0,0 +1,368 @@
+        fs::write(path, json)?;
+        Ok(())
+    }
+}
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let cli = Cli::parse();
+
+    match cli.command {
+        Commands::GenerateKeys { secret_keys_file } => {
+            let enc_sk = PrivateKey::<G2Element>::new(&mut thread_rng());
+            let enc_pk = PublicKey::<G2Element>::from_private_key(&enc_sk);
+            let signing_sk = G2Scalar::rand(&mut thread_rng());
+            let signing_pk = G2Element::generator() * signing_sk;
+
+            let enc_pk_hex = Hex::encode_with_format(bcs::to_bytes(&enc_pk)?);
+            let signing_pk_hex = Hex::encode_with_format(bcs::to_bytes(&signing_pk)?);
+            let enc_sk_hex = Hex::encode_with_format(bcs::to_bytes(&enc_sk)?);
+            let signing_sk_hex = Hex::encode_with_format(bcs::to_bytes(&signing_sk)?);
+
+            // Write secret keys to file
+            let secret_keys_content =
+                format!("enc_sk: {}\nsigning_sk: {}\n", enc_sk_hex, signing_sk_hex);
+
+            if let Some(parent) = secret_keys_file.parent() {
+                fs::create_dir_all(parent)?;
+            }
+
+            write_secret_file(&secret_keys_file, &secret_keys_content)?;
+
+            println!("Secret keys written to: {}", secret_keys_file.display());
+            #[cfg(not(unix))]
+            println!("WARNING: On non-Unix systems, manually restrict file permissions");
+            println!();
+            println!("==============STEP 1: REGISTER ONCHAIN===============");
+            println!("export MY_ADDRESS=<your address to use> # check with: sui client addresses");
+            println!("export DKG_ENC_PK={}", enc_pk_hex);
+            println!("export DKG_SIGNING_PK={}", signing_pk_hex);
+            println!();
+            println!("sui client switch --address $MY_ADDRESS");
+            println!("sui client call --package $COMMITTEE_PKG --module seal_committee \\");
+            println!("  --function register \\");
+            println!("  --args $COMMITTEE_ID x\"$DKG_ENC_PK\" x\"$DKG_SIGNING_PK\" \"https://your-url.com\"");
+            println!();
+            println!(
+                "==============STEP 2: AFTER ALL MEMBERS REGISTERS, RUN DKG INIT ==============="
+            );
+            println!(
+                "export DKG_ENC_SK=$(grep enc_sk {} | cut -d: -f2 | xargs)",
+                secret_keys_file.display()
+            );
+            println!(
+                "export DKG_SIGNING_SK=$(grep signing_sk {} | cut -d: -f2 | xargs)",
+                secret_keys_file.display()
+            );
+            println!();
+            println!("cargo run dkg-cli init --my-address $MY_ADDRESS --committee-id $COMMITTEE_ID --state-dir ./state");
+        }
+
+        Commands::Init {
+            my_address,
+            committee_id,
+            key_server_id,
+            network,
+            state_dir,
+        } => {
+            // Read secrets from environment variables.
```

Comment:
why not load from the file?

Reply:
revised to:
1. generate-keys: creates .dkg.key containing 2 sk and 2 pk. use export for 2 pks for register command. 
2. init now accepts a keys file path, deserialize 2 sks and initialize the /dkg-state file.